packages/analytics-js/src/constants/consent.ts

```diff
@@ -0,0 +1,3 @@
+const SUPPORTED_CONSENT_MANAGERS = ['oneTrust'];
```

Comment:
Define this as an enum and use it in the `CookieConsentOptions` type.

Reply:
Addressed